src/CDPSession.re

```diff
@@ -0,0 +1,6 @@
+type t;
+
+[@bs.send] external detach : t => Js.Promise.t(unit) = "";
+
+[@bs.send]
+external send : (t, string, Js.t({..})) => Js.Promise.t(Js.t({..})) = "";
```

Comment:
The third argument `params` is optional.

Reply:
Fixed in commit a980bc6